migrator/clone/db_clone_manager_impl.go

```diff
@@ -126,7 +136,7 @@ func (d *dbCloneManagerImpl) Persist(cloneName string, pgClone string, persistBo
 
 			// If the versions do not match, we have updated another time with Postgres,
 			// so we can no longer roll back to RocksDB.
-			if rocksVersion == nil || (currentPostgresVersion != nil && rocksVersion.MainVersion != currentPostgresVersion.MainVersion) {
+			if rocksVersion != nil && currentPostgresVersion != nil && rocksVersion.MainVersion != currentPostgresVersion.MainVersion {
```

Comment:
I do not quite understand this change. If rocksVersion is nil, likely Rocksdb current does not exist. The new codes are for sure correct. But why did we decommission in this case before?

Reply:
@c-du The previous idea was that if rocksDB was empty we could go ahead and clean it up.  That was really an unnecessary assumption and causes issues when testing things like this.  And this logic is a little cleaner.

Reply:
@c-du The previous idea was that if rocksDB was empty we could go ahead and clean it up.  That was really an unnecessary assumption and causes issues when testing things like this.  And this logic is a little cleaner.